script/generate-documentation

```diff
@@ -103,14 +102,24 @@ SSH_IV=$2
 
 check_asciidoctor() {
     asciidoctor_bin=$(command -v asciidoctor) || true
-    if [[ -z $asciidoctor_bin ]] || [[ ! -f $asciidoctor_bin ]]; then
+    [[ -n "$asciidoctor_bin" ]] || \
+        for asciidoctor_bin in "${GEM_HOME}"/bin/asciidoctor.ruby*; do :; done
+
+    if [[ -z "$asciidoctor_bin" ]] || [[ ! -f $asciidoctor_bin ]]; then
         echo "Could not find asciidoctor binary in your path, please install it and run this command again:"
         echo "    sudo gem install asciidoctor pygments.rb"
         echo "    sudo gem install asciidoctor-pdf --pre"
         echo "    sudo zypper install ruby2.6-rubygem-asciidoctor"
         echo "    sudo zypper install 'perl(Pod::AsciiDoctor)'"
         exit 1
     fi
+
+    command -v asciidoctor-pdf || [ -s /usr/bin/asciidoctor-pdf ] || {
+        local asciidoctor_pdf_bin
+        for asciidoctor_pdf_bin in "${GEM_HOME}"/bin/asciidoctor-pdf.ruby*; do :; done
+        [ -z "$asciidoctor_pdf_bin" ] || \
+            sudo ln "$asciidoctor_pdf_bin" /usr/bin/asciidoctor-pdf
```

Comment:
The `sudo` seems a bit problematic. Keep in mind that this script should also work locally where writing to `/usr/bin/` is not desirable.

Reply:
Yeah it looks this part is not needed for CI, so I've removed it